### PR TITLE
Debugging hive and hadoop on EMR

### PR DESCRIPTION
**Pull request by eduardog**
https://thinkneardev.atlassian.net/browse/REX-2756

### CODE REVIEW

File: source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Debugging Hadoop 2.x on Amazon EMR"
+date: 2015-05-11 13:48:06 -0700
+author: Eduardo Gonzalez
+comments: true
+is_newest: true
```
Comment (eduardog) [inline source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown]: we have two open PRs right now for blog posts.  I guess I'll rebase mine to remove the flag off of Tassos's post after his has gone up?

Comment (eduardog) [inline source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown]: this has now been done.

**Comment by eduardog** [comment]
Rebased on top of Tassos' post and ready for review.

@thinknearken, it looks like you accidentally merged `master` into `source` on 2015-05-18.  We should probably fix that before merging this, because I'm pretty sure those source trees are supposed to stay separate?

**Comment by eduardog** [comment]
Tried another markdown-to-google-doc conversion for commenting:
`pandoc -fmarkdown_github --tab-stop=2 -tdocx`
https://docs.google.com/a/thinknear.com/document/d/1fVIf7qZIrZNSpR9-dkq6n7fbH2eDwQEJ-tdRWBA4BlE/edit?usp=sharing

File: source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    <match enabled="true" name="10.*" pattern="http://10.*" isRegEx="false" isBlackList="false" isMultiLine="false" caseSensitive="false" fromSubscription="false" />
+                    <match enabled="true" name="ec2.internal" pattern="*ec2.internal:*" isRegEx="false" isBlackList="false" isMultiLine="false" caseSensitive="false" fromSubscription="false" />
+                </matches>
+                <manualconf host="localhost" port="8157" socksversion="5" isSocks="true" username="" password="" domain="" />
+            </proxy>
+        </proxies>
+    </foxyproxy>
+
+### Useful Hadoop Interface Pages
+{*TODO* improve subheading name?}
```
Comment (thinknearken) [inline source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown]: Can you just get rid of this line

Comment (thinknearken) [inline source/_posts/2015-05-11-debugging-hive-0-dot-13-on-amazon-emr.markdown]: line 42 only that is... the TODO

**Comment by eduardog** [comment]
Updated. Do we want to fix upstream (`master`, the published data, was merged into `source`, the branch we commit to for octopress to convert into `master`) or was that intentional?

**Pull request merged**